apple/RNCWebViewImpl.m

```diff
@@ -246,29 +252,51 @@ - (void)startLongPress:(UILongPressGestureRecognizer *)pressSender
     if (pressSender.state != UIGestureRecognizerStateEnded || !self.menuItems) {
         return;
     }
-    // When a long press ends, bring up our custom UIMenu if defined
-    if (self.menuItems.count == 0) {
+    if (@available(iOS 16.0, *)) {
+      CGPoint location = [pressSender locationInView:self];
+      UIEditMenuConfiguration *config = [UIEditMenuConfiguration configurationWithIdentifier:nil sourcePoint:location];
+      [_editMenuInteraction presentEditMenuWithConfiguration:config];
+    } else {
+      // When a long press ends, bring up our custom UIMenu if defined
```

Comment:
If we need to support older versions of Xcode, I can put this behind a compiler macro like we do elsewhere, along with any other references.